qcengine/config.py

```diff
@@ -27,6 +27,8 @@ def get_global(key: Optional[str] = None) -> Union[str, Dict[str, Any]]:
     import cpuinfo
     import psutil
 
+    # TODO (wardlt): Implement a means of getting CPU information from compute nodes on clusters
```

Comment:
Can you describe what is missing here? `cpuinfo` pulls quite a bit of information.

Reply:
The amount of information is fine. However, the cpuinfo is pulled from batch node and not the compute node if QCEngine is configured for MPI tasks. This means the provenance information is not for the CPU that actually ran the calculation

Reply:
Ah, I see and makes sense. Can you add a further comment here to explain this?

Reply:
Sure